hw/ip/otbn/data/otbn_sec_cm_testplan.hjson

```diff
@@ -25,119 +25,172 @@
+               local escalation to the locked state.
+            '''
       stage: V2S
-      tests: ["otbn_imem_err", "otbn_dmem_err", "otbn_zero_state_err_urnd", "otbn_illegal_mem_acc"]
+      tests: ["otbn_imem_err", "otbn_dmem_err", "otbn_zero_state_err_urnd", "otbn_illegal_mem_acc",
+              "otbn_sec_cm"]
     }
     {
       name: sec_cm_controller_fsm_sparse
-      desc: "Verify the countermeasure(s) CONTROLLER.FSM.SPARSE."
+      desc: '''Verify the countermeasure(s) CONTROLLER.FSM.SPARSE.
+            This countermeasure is verified with a standardized test.
+            '''
       stage: V2S
       tests: ["otbn_sec_cm"]
     }
     {
       name: sec_cm_scramble_key_sideload
-      desc: "Verify the countermeasure(s) SCRAMBLE.KEY.SIDELOAD."
+      desc: '''Verify the countermeasure(s) SCRAMBLE.KEY.SIDELOAD
+
+            Simulation can't really prove that the sideload key is unreachable by SW.
+            However, from defined CSRs and memory returned data, there is no way to read
+            scramble key by SW.
+            '''
       stage: V2S
       tests: ["otbn_single"]
     }
     {
       name: sec_cm_scramble_ctrl_fsm_local_esc
-      desc: "Verify the countermeasure(s) SCRAMBLE_CTRL.FSM.LOCAL_ESC."
+      desc: '''Verify the countermeasure(s) SCRAMBLE_CTRL.FSM.LOCAL_ESC.
+            The controller FSM moves to a terminal error state upon local escalation.
+            1. IMEM/DMEM error tests to see local escalation related with integrity Checking
+            2. Zero state URND test to see local escalation regarding a URND value of all zeros
+            3. Illegal memory access test to see local escalation while having illegal read and
+               write accesses to the IMEM when the OTBN is busy.
+            4. Bad internal state errors that are triggered by otbn_sec_cm test will also cause
+               local escalation to the locked state.
+            '''
       stage: V2S
-      tests: ["otbn_imem_err", "otbn_dmem_err", "otbn_zero_state_err_urnd", "otbn_illegal_mem_acc"]
+      tests: ["otbn_imem_err", "otbn_dmem_err", "otbn_zero_state_err_urnd", "otbn_illegal_mem_acc",
+              "otbn_sec_cm"]
     }
     {
       name: sec_cm_scramble_ctrl_fsm_sparse
-      desc: "Verify the countermeasure(s) SCRAMBLE_CTRL.FSM.SPARSE."
+      desc: '''Verify the countermeasure(s) SCRAMBLE_CTRL.FSM.SPARSE.
+            This countermeasure is verified with a standardized test.
+            '''
       stage: V2S
       tests: ["otbn_sec_cm"]
     }
     {
       name: sec_cm_start_stop_ctrl_fsm_global_esc
-      desc: "Verify the countermeasure(s) START_STOP_CTRL.FSM.GLOBAL_ESC."
+      desc: '''Verify the countermeasure(s) START_STOP_CTRL.FSM.GLOBAL_ESC.
+
+            '''
       stage: V2S
       tests: ["otbn_escalate"]
     }
     {
       name: sec_cm_start_stop_ctrl_fsm_local_esc
       desc: "Verify the countermeasure(s) START_STOP_CTRL.FSM.LOCAL_ESC."
```

Comment:
Don't we want a similar expansion of the description as we had for SCRAMBLE_CTRL.FSM.LOCAL_ESC?